ClientApp/package.json

```diff
@@ -19,9 +19,10 @@
     "@angular/platform-browser-dynamic": "~10.1.1",
     "@angular/router": "~10.1.1",
     "@fortawesome/angular-fontawesome": "^0.7.0",
+    "@fortawesome/fontawesome-common-types: "^0.2.32",
```

Comment:
Codacy found an issue: [Unexpected character ('^' (code 94)): was expecting a colon to separate field name and value](https://app.codacy.com/gh/InvestingOak/InvestingOak/pullRequest?prid=6289442)